models/location.js

```diff
@@ -11,5 +11,20 @@ module.exports = (sequelize, DataTypes) => {
   Location.associate = function(models) {
```

Comment:
so this looks great, however if the issue is getting the return value of the forecast function is not returning properly, i would suggest doing under line 16 return new promise function(resolve, reject) and encapsulate the entire fetch call in your promise. then on like 22 and 25 for return response or return error, do resolve(response) and reject(error), and then make sure you treat the function as async when you calll it

Reply:
Thanks, Ty! That syntax did the trick. I should be able to implement the same thing other places in my project as well.